src/VuePhoneNumberInput/index.vue

```diff
@@ -114,10 +117,13 @@
       },
       locale () {
         const locale = this.defaultCountryCode || (!this.noUseBrowserLocale ? browserLocale() : null)
-        const countryAvailable = isCountryAvailable(locale)
+        const countryAvailable = countries.find(function(element) {
+          return element.iso2 === locale;
+        });
 
         if (countryAvailable && locale) {
-          this.countryCode = locale
+          this.countryCode = locale;
+          this.countryName = this.defaultCountryName || countryAvailable.name;
```

Comment:
Here should just be this.countryName = countryAvailable.name;
Can put the this.countryName = this.defaultCountryName in the else if